src/components/OpenGovTreasuryProposal/WriteProposal.tsx

```diff
@@ -264,13 +265,27 @@ const WriteProposal = ({
 										name='title'
 										className='h-[40px] rounded-[4px] dark:border-separatorDark dark:bg-transparent dark:text-blue-dark-high dark:focus:border-[#91054F]'
 										onChange={(e) => {
-											setTitle(e.target.value);
-											onChangeLocalStorageSet({ title: e.target.value }, Boolean(isDiscussionLinked));
-											setSteps({ percent: content.length === 0 ? 83.33 : 100, step: 0 });
+											const value = e.target.value;
```

Comment:
please try to use form Item validation like 	rules={[
									{
										message: `Please add a valid discussion link for ${network} Network`,
										validator(rule, value, callback) {
											if (callback && isDiscussionLinkedValid(value)) {
												callback(rule?.message?.toString());
											} else {
												callback();
											}
										}
									}
								]}